contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -221,9 +221,9 @@ contract VaultAdmin is VaultStorage {
             require(strategyTo.supportsAsset(_assets[i]), "Asset unsupported");
             // Withdraw from Strategy and pass other Strategy as recipient
             strategyFrom.withdraw(address(strategyTo), _assets[i], _amounts[i]);
-            // Tell new Strategy to deposit into protocol
-            strategyTo.deposit(_assets[i], _amounts[i]);
         }
+        // Tell new Strategy to deposit into protocol
+        strategyTo.depositAll();
```

Comment:
We should include VaultAdmin in deploy 14

Reply:
Oops yes I will need to update. I think when I wrote the deploy script I was thinking we could sneak it in with that other upgrade which had all these things. Will update 👍